### PR TITLE
Update INSTALL - improve installation w/ configure

### PR DESCRIPTION
**Pull request by sarpik**
I was building from the github repo and found it weird that the `configure` file was not present.
Only randomly I happened to notice that the `make configure` is what generates it.

I know I should've read the whole thing, but sometimes you might not have enough time and/or you just want things to work, thus I made this change.

(I tried moving up the `Note` part, but it looked kinda out-of-place, so I guess you could experiment with it yourself so that it's not annoying for those who build from the release but those who build from source see it before actually building).

### CODE REVIEW

**Comment by jtheoof** [comment]
There is an `autogen.sh` to generate the file. Seems pretty standard when using `autoconf`. See [the doc](https://www.gnu.org/software/automake/faq/autotools-faq.html#What-does-_002e_002fbootstrap-or-_002e_002fautogen_002esh-do_003f)

File: INSTALL.adoc [hunk 1]
```diff
@@ -43,6 +43,7 @@ Installation using `configure`
 
 Optionally, you can use the `configure` script to detect dependencies:
 
+	$ make configure	# if you're building from the Tig repository (requires `autoconf`)
```
Comment (jtheoof) [inline INSTALL.adoc]: `./autogen.sh`

Comment (sarpik) [reply INSTALL.adoc]: Alright, thanks for the info @jtheoof. Should I change the line to `$ ./autogen.sh` or do nothing and just close this PR?

Comment (jtheoof) [reply INSTALL.adoc]: No I would keep the MR but using my recommended fix.

Perhaps @jonas will merge. It's fairly easy to review.

Comment (sarpik) [reply INSTALL.adoc]: If this is what you've meant - I've changed it to `./autogen.sh`.

**Comment by jonas** [comment]
I ended up moving the note about running `make configure` up. Maybe it won't get read as you said. I will think about whether to split the section into "building with configure from git source" and "build from release tarball".

Thanks for the PR!

**Pull request closed without merging**